src/main/java/net/sf/jabref/JabRef.java

```diff
@@ -96,6 +99,26 @@ public void start(String[] args) {
             System.setProperty("proxySet", "true");
         }
 
+        Authenticator.setDefault(new Authenticator() {
+
+            @Override
+            protected PasswordAuthentication getPasswordAuthentication() {
+                if (getRequestorType() == RequestorType.PROXY) {
+                    String prot = getRequestingProtocol().toLowerCase();
+                    String host = System.getProperty(prot + ".proxyHost", "");
+                    String port = System.getProperty(prot + ".proxyPort", "80");
+                    String user = System.getProperty(prot + ".proxyUser", "");
+                    String password = System.getProperty(prot + ".proxyPassword", "");
+                    if (getRequestingHost().equalsIgnoreCase(host)) {
+                        if (Integer.parseInt(port) == getRequestingPort()) {
+                            return new PasswordAuthentication(user, password.toCharArray());
+                        }
+                    }
+                }
+                return null;
+            }
+        });
```

Comment:
Is the `Authenticator` always needed? Can we add it only in the case we really know for sure that username/password credentials are required?